test/units/vsoClientTests.ts

```diff
@@ -0,0 +1,149 @@
+import assert = require('assert');
+import nock = require('nock');
+import vsom = require('../../api/VsoClient');
+import * as rm from 'typed-rest-client/RestClient';
+import { resolve } from 'path';
+describe('VSOClient Units', function () {
+    let rest: rm.RestClient;
+    let vsoClient: vsom.VsoClient
+    const baseUrl = 'https://dev.azure.com/';
+    before(() => {
+        const userAgent = "testAgent";
+        const handlers = [];
+        const options = null;
+        rest = new rm.RestClient(userAgent, null, handlers, options);
+        vsoClient = new vsom.VsoClient(baseUrl, rest);
+    });
+
+    after(() => {
+    });
+
+    afterEach(() => {
+        nock.cleanAll();
+        vsoClient = new vsom.VsoClient(baseUrl, rest);
+    });
+
+    it('constructs', () => {
+        this.timeout(1000);
```

Comment:
Can we organize  these using Arrange, Act, Assert?(as commented sections e.g. // Arrange .... // Act ... // Assert